assets/js/components/setup/CompatibilityChecks/checks.js

```diff
@@ -29,14 +30,31 @@ import {
 	ERROR_WP_PRE_V5,
 } from './constants';
 
+const isIP = /^(?:25[0-5]|2[0-4]\d|1\d\d|[1-9]\d|\d)(?:\.(?:25[0-5]|2[0-4]\d|1\d\d|[1-9]\d|\d)){3}$/;
+
+const invalidTlds = /\.(example|invalid|localhost|test)$/;
+
+const invalidIpRanges = [
+	{ subnet: '10.0.0.0', mask: 8 },
+	{ subnet: '127.0.0.0', mask: 8 },
+	{ subnet: '172.16.0.0', mask: 12 },
+	{ subnet: '192.168.0.0', mask: 16 },
+];
```

Comment:
These are specific to IP range checks and not generic compatibility check values. Let's move them to their own file as they aren't necessarily specific to this component. Moving them to `assets/js/util/ip-cidr.js` would make sense to me.

Also, quite minor, but please be consistent with the `IP` capitalisation. `invalidIpRanges` should be `invalidIPRanges`, similar to `isIP`. Since "TLDs" is an acronym too, let's use `invalidTLDs`.

Reply:
I guess maybe these are actually pretty tightly-coupled to this hostname check… so if you think it's better to leave these in here that's fine. Up to you, but either way let's make the capitalisation consistent 🙂